data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+
+substitutions
+	<EDC-ship> "Esketari"
+
+mission "Efreti: Distress Call 1"
+	name "Investigate distress call"
+	description "The Quarg have asked you investigate an Efreti distress call in <destination>."
+	source "Karek Fornati"
+	stopover "Seleptra Nak"
+	destination "Laki Nemparu"
+	passengers 40
+	landing
+	minor
+	random < 30
+	blocked "A Quarg ship broadcasts that they need a ship with at least <bunks> bunks for an emergency evacuation."
+
+	to offer
+		not "event: wanderers: kor mereti friendly"
+		has "First Contact: Karek Fornati: offered"
+		
+	on offer
+		conversation
+			`The blue flash of Quarg ships is nothing new as you travel Efreti space. Usually that is all you see of the enigmatic aliens. Today, however, you receive an urgent hail as you land.`
+			`	"Salutations Captain of the <ship>. Your most diligent efforts are required lest many families mourn the dead. An Efreti ship, the <EDC-ship>, was surveying <stopovers> when surprised by Mereti they were. Their ship crashed on the planet with many dead and they have asked for help to evacuate."`
+			choice
+				`	"I'll go there immediately."`
+					accept
+				`	"Can't the Quarg respond?"`
+					goto quarg
+				`	"I don't care about the Korath."`
+					goto careless
+			
+			label quarg
+			`	"We fought off the Mereti that attacked them but they had crashed before we could reach them. The gravity on that world requires us to have hardened environment suits to navigate wreckage while a Human would have few problems. You can reach them days before we could return from Kuwaru Efreti."`
+			choice
+				`	"I'll go help, then."`
+					accept
+				`	"I won't risk my ship for the Korath."`
+					goto careless
+
+			label careless
+			`	The Quarg is silent for some time. Eventually it responds, "Life is rare and precious across the vastness of space. Look about the Korath worlds and see what remains when one cares not what others suffer."`
```

Comment:
```suggestion
			`	The Quarg is silent for some time. Eventually, it responds, "Life is rare and precious across the vastness of space. Look about the Korath worlds and see what remains when one cares not what others suffer."`
```

> This is a strange thing to say: the Korath did that to themselves.
If it means that no one helped them, it's a rather damning indictment of the Quarg's own actions, or lack thereof, in that time.

This can also refer to the Korath not caring about others within their own race, right? I think it makes sense in that context.